tests/test_skvbc_fast_path.py

```diff
@@ -43,33 +43,31 @@ def setUp(self):
 
     @with_trio
     @with_bft_network(start_replica_cmd)
-    async def test_fast_path_read_your_write(self, bft_network):
+    @verify_linearizability
+    async def test_fast_path_read_your_write(self, bft_network, tracker):
         """
         This test aims to check that the fast commit path is prevalent
         in the normal, synchronous case (no failed replicas, no network partitioning).
 
-        First we write a series of known K/V entries.
+        First we write a series of K/V entries and tracked them using the tracker from the decorator.
         Then we check that, in the process, we have stayed on the fast path.
 
-        Finally we check if a known K/V has been executed.
+        Finally the decorator verifies the KV execution.
         """
         bft_network.start_all_replicas()
-        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
-
         for _ in range(10):
-            key, val = await skvbc.write_known_kv()
+            await tracker.run_concurrent_ops(num_ops=1, write_weight=0.7)
```

Comment:
why not remove the for loop here too ?

Reply:
Just inattention because this test is the simplest and always passed so I forgot to handle it